package.json

```diff
@@ -26,7 +26,7 @@
     "build:profile": "rm -rf build && cp envConfig/local.env .env && IS_PROD_PROFILE=true NODE_ENV=production webpack",
     "build:live": "cp envConfig/live.env .env && NODE_ENV=production webpack",
     "build:live:debug": "rm -rf build && awk '{sub(/LOG_DIR=.+/,\"LOG_DIR='log'\")}1' envConfig/live.env > .env && NODE_ENV=production webpack",
-    "build:storybook": "node .storybook/buildFontPreloads && build-storybook -s .storybook/static -c .storybook -o storybook_dist",
```

Comment:
is this script still used for anything after being removed from here?  could probably also be removed from the file system

Reply:
oh yeh, I'll remove the script too. thanks